Nos/Service/Relay/RelayService.swift

```diff
@@ -381,12 +379,16 @@ extension RelayService {
             return false
         } else {
             let remainingEventCount = await parseQueue.count
-            try await self.parseContext.perform {
+            try await persistenceController.parseContext.perform {
                 var savedEvents = 0
                 for (event, socket) in eventData {
-                    let relay = self.relay(from: socket, in: self.parseContext)
+                    let relay = self.relay(from: socket, in: self.persistenceController.parseContext)
                     do {
-                        if try EventProcessor.parse(jsonEvent: event, from: relay, in: self.parseContext) != nil {
+                        if try EventProcessor.parse(
```

Comment:
I'm fine from switching to self.persistenceController.parseContext  but I prefer it to be in just one line. For some reason, I never got used to multiline `if` clauses. If the line limit is an issue, solutions could be storing the result of .parse() in a variable just one line above would do or storing self.persistenceController.parseContext in a variable.